backend/tests/unit/services/test_alert_service.py

```
@@ -24,48 +26,42 @@ class TestAlertServiceInitialization:
+            assert service.runbook_service == mock_runbook.return_value
+            assert service.history_service == mock_history.return_value
+            assert service.chain_registry == mock_chain_registry.return_value
+            assert service.mcp_server_registry == mock_mcp_registry.return_value
+            assert service.mcp_client == mock_mcp_client.return_value
+            assert service.llm_manager == mock_llm_manager.return_value
+            assert service.agent_factory is None  # Not initialized yet
     
     def test_initialization_with_dependencies(self, mock_settings):
         """Test that dependencies are created with correct parameters."""
         with patch('tarsy.services.alert_service.RunbookService') as mock_runbook, \
              patch('tarsy.services.alert_service.get_history_service') as mock_history, \
```

Ruff (0.12.2)

Local variable `mock_history` is assigned to but never used

Remove assignment to unused variable `mock_history`

(F841)